spaces/aliases.json

```
@@ -52,4 +52,5 @@
   "dforce": "dforcenet.eth",
   "rari": "fuse.eth",
   "nft20.eth": "musedao.eth"
+  "tripscommunity": "tripscommunity.eth"
```

A comma is missing at the end of the line above your space.
```suggestion
  "nft20.eth": "musedao.eth",
  "tripscommunity": "tripscommunity.eth"
```